src/components/ui/TopBar.vue

```diff
@@ -1,15 +1,20 @@
 <template>
   <header class="top-bar" :style="{ marginBottom: extraPadding + 'px' }">
     <div class="top-bar__bar">
-      <RouterLink :to="{ name: 'Home' }" class="top-bar__link top-bar__link--logo">
+      <RouterLink
+        :to="{ name: 'Home' }"
+        class="top-bar__link top-bar__link--logo"
+      >
         <img src="@/assets/images/mozilla.svg" :alt="fluent('mozilla')" />
       </RouterLink>
       <SearchForm
+        v-if="loggedIn"
```

Comment:
There is a story for removing the search box if the user is not logged in?

Reply:
As per today's call: hiding the search bar for now is part of gradually shipping this.